bitnami/minio/templates/api-ingress.yaml

```diff
@@ -0,0 +1,57 @@
+{{- if .Values.apiIngress.enabled -}}
+apiVersion: {{ include "common.capabilities.ingress.apiVersion" . }}
+kind: Ingress
+metadata:
+  name: {{ include "common.names.fullname" . }}
```

Comment:
Maybe there need change to different name, because this name is used by ingress.yaml? currently have a error when upgrade it.

```
[09:33:11.979274] PATH:
[09:33:11.979608]   /usr/bin/helm
[09:33:11.979895] 
[09:33:11.980177] ARGS:
[09:33:11.980504]   0: helm (4 bytes)
[09:33:11.980801]   1: upgrade (7 bytes)
[09:33:11.981176]   2: --install (9 bytes)
[09:33:11.981504]   3: --reset-values (14 bytes)
[09:33:11.981796]   4: minio (5 bytes)
[09:33:11.982064]   5: bitnami/minio (13 bytes)
[09:33:11.982391]   6: --version (9 bytes)
[09:33:11.982684]   7: ^8.1.1 (6 bytes)
[09:33:11.983001]   8: --create-namespace (18 bytes)
[09:33:11.983322]   9: --namespace (11 bytes)
[09:33:11.983633]   10: production (10 bytes)
[09:33:11.983922]   11: --values (8 bytes)
[09:33:11.984215]   12: /tmp/helmfile219781721/production-minio-values-7bc486444b (57 bytes)
[09:33:11.984501]   13: --history-max (13 bytes)
[09:33:11.984836]   14: 10 (2 bytes)
[09:33:11.985124] 
[09:33:11.985429] ERROR:
[09:33:11.985790]   exit status 1
[09:33:11.986119] 
[09:33:11.986399] EXIT STATUS
[09:33:11.986712]   1
[09:33:11.987031] 
[09:33:11.987317] STDERR:
[09:33:11.987643]   Error: ingresses.networking.k8s.io "minio" already exists
[09:33:11.987934] 
[09:33:11.988269] COMBINED OUTPUT:
[09:33:11.988565]   Release "minio" does not exist. Installing it now.
[09:33:11.988855]   Error: ingresses.networking.k8s.io "minio" already exists
```